_episodes/03-understanding-yaml.md

```diff
@@ -29,7 +29,7 @@ YAML (Yet Another Markup Language or sometimes popularly referred to as YAML Ain
 
 ~~~
 number-value: 42
-floating-point-value: 3.141592
```

Comment:
bit nitpicky, but this is what I would expect. For a floating point `1.10 == 1.1`. The issue with the python versions is that they are _not_ floating numbers yet are specified as such. So let's perhaps leave this line as it is and add another line "python-verion: 3.10" and then add the callout for this?

Reply:
And even 1.1 is not actually 1.1 ;-) https://www.h-schmidt.net/FloatConverter/IEEE754.html.

I would be careful with putting line in the example listing which have incorrect behavior, even if we call attention to it later. If we put in a python-version line, we should do it was a string `'3.10'`, not as a float.

Reply:
Thanks for the review! I've pushed another commit with the required modifications.